test/functional/services/filter_bar.js

```diff
@@ -23,15 +23,23 @@ export function FilterBarProvider({ getService }) {
       await testSubjects.click(`filter & filter-key-${key} disableFilter-${key}`);
     }
 
-    async addFilter(field, operator, value) {
+    /**
+     *
+     * @param field
+     * @param operator
```

Comment:
Same here. The comment block is fairly useless except for the `inputType` clarification, IMO.

Reply:
Agree on this one, I'll remove. not to mention the variable name is wrong :).